centos.org/pipelines/lib/foremanCentosJob.groovy

```diff
@@ -25,10 +25,14 @@ pipeline {
             steps {
                 script {
                     if (params.type == 'pulpcore') {
-                        setup_extra_vars = ['forklift_install_pulp_from_galaxy': true, 'forklift_install_from_galaxy': false]
+                        if (pulpcore_version >= '3.18'){
+                            setup_extra_vars = ['forklift_install_pulp_from_galaxy': true, 'forklift_install_from_galaxy': false, 'pipeline_version' : '3.18']
```

Comment:
why not passing `params.version` directly, or `pulpcore_version`? instead of hardcoding it here.

Reply:
Yeah, I totally forgot about params